### DISCUSSION TITLE
Use page navigation in a long blog with long TOC

### DISCUSSION THREAD

**Discussion by ning3739** [quarto-dev/quarto-cli#5361] [Q&A] [answered]
I'm facing an issue where my [blog](https://datalabzone.com/series/r/2022/12/01/datavisualization) content is too long, which has resulted in a lengthy table of contents. Despite providing titles for each section, it still makes it difficult for readers to navigate. Therefore, I would like to ask if it's possible for me to divide this long blog into smaller sections and merge them together using page navigation, so that they can be displayed within the same blog post? Thank you.
![Screenshot 2023-04-30 at 8 14 48 PM](https://user-images.githubusercontent.com/94179273/235343544-016c3640-1296-46be-b108-8d850f56f66a.png)
![Screenshot 2023-04-30 at 8 16 33 PM](https://user-images.githubusercontent.com/94179273/235343550-e2e9ac17-4dd1-48e4-9e96-7382b17d0795.png)

**Comment by ning3739** [comment]
If it possible, how can i do it? thanks

**Comment by mcanouil** [comment, marked as answer]
The TOC is for the blog post, even if you split your main document into smaller pieces and then you "merge" together the pieces using the include short code, the results will remain the same.

What you can do, shorten your titles which is one of the reason for the long TOC, _i.e._, you have a narrow sidebar and long titles which appear as multiline.
In addition to shorter titles for sections, you can increase the width of the sidebar, see <https://quarto.org/docs/output-formats/page-layout.html#grid-customization>.

Personal note: you have an introduction section which has an introduction subsection, and you also have another introduction section. In my opinion, that's a lot of "introductions" 😅 It starts to feel you were writing a book with chapters and not a blog post.

**Reply by ning3739** [reply]
Thank you so much!
